### DISCUSSION TITLE
Live Overlay

### DISCUSSION THREAD

**Discussion by priestkos** [Glazelf/RiseHax#6] [Ideas]
Once the game is released, a possible live overlay (edit: section) in the app for the following will be much appreciated: 

1. Monster HP, 
2. Weapon attack stat, 
3. Monster partbreak threshold, 
4. Monster stagger threshold, 
5. Wyvern riding threshold, 
6. Wyvern riding cooldown, 
7. Silkbind attacks/aerial attacks performed, 
8. Monster spawned is a crown size or not.

**Comment by Glazelf** [comment]
By overlay do you mean a screen on the Switch itself or in the application window?  
We do have this section (displayed in red below) reserved for displaying monster information where we could integrate the attributes you mentioned.   
I don't think creating an overlay on the Switch itself would be possible using a PC application, someone would have to look into creating an [Edizon](https://github.com/WerWolv/EdiZon) overlay.
![RiseHax Quest window](https://i.imgur.com/7SdF9nb.png)

**Reply by priestkos** [reply]
Yes, I meant a screen in the application window. The screen you showed here is a good place to integrate everything for sure.
